templates/etc/ssh/sshd_config.j2

```diff
@@ -133,3 +133,12 @@ Match Group sftp-only
     AllowTcpForwarding no
     X11Forwarding no
     #PermitTTY no
+{% if ssh_match_blocks | d() %}
```

Comment:
Make `d(False)` instead of only `d()`, it will be clearer what happens if nothing is defined.

Reply:
```suggestion
{% if ssh_match_blocks | d(False) %}
```